atlasdb-config/src/test/java/com/palantir/atlasdb/http/RedirectRetryTargeterTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.http;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import org.junit.Test;
+
+import com.google.common.collect.ImmutableList;
+
+public class RedirectRetryTargeterTest {
+    private static final URL URL_1;
+    private static final URL URL_2;
+    private static final URL URL_3;
+
+    static {
```

Comment:
*sniff 😢

Reply:
It might be cleaner to have a helper method `createURL` which would deal with the exceptions. That way we could remove the static block.

Reply:
:+1: Yep, will use @OStevan's suggestion